src/main/java/teammates/common/util/Const.java

```diff
@@ -1415,6 +1417,9 @@ private Const() {
 
         // HTTP parameter null message
         public static final String NULL_HTTP_PARAMETER = "The [%s] HTTP parameter is null.";
+
+        // Response Body null message
+        public static final String INVALID_REQUEST_BODY = "%s cannot be null";
```

Comment:
- `Invalid` and `Null` are not necessarily one and the same.
- This doesn't seem to be used anywhere?

Reply:
Yeap my bad.. I initially added a couple of consts in my initial submission of this PR (but eventually while refactoring they became unused). Have removed